packages/internal/src/generate/clientPreset.ts

```diff
@@ -32,27 +32,6 @@ export const generateClientPreset = async () => {
     schema: getPaths().generated.schema,
     documents: documentsGlob,
     generates: {
-      // should be graphql.d.ts
```

Comment:
Why can you remove this now? How is this related to just adding `| TypedDocumentNode<TData, TVariables>` to some types in `packages/web/src/components/GraphQLHooksProvider.tsx`?

Reply:
It's not related to this PR, it was an extraneous comment that no longer applied and I removed the comment. As can see I removed the `      [`${getPaths().web.base}/types/types.d.ts`]: ` which I thought was needed but realized that preset in fact created the proper files.

This was just preset cleanup -- I guess I could have made a separate PR but as I gather I simply combined it as notice no longer needed.